src/Connectors/test/Connector.EFCore.Test/Steeltoe.Connector.EFCore.Test.csproj

```diff
@@ -21,7 +21,7 @@
   <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp3.1'">
     <PackageReference Include="MySql.EntityFrameworkCore" Version="$(MySqlEFCoreVersion)" />
   </ItemGroup>
-  <ItemGroup Condition="'$(TargetFramework)' == 'net6.0'">
+  <ItemGroup Condition="'$(TargetFramework)' == 'net8.0' Or '$(TargetFramework)' == 'net6.0'">
```

Comment:
Intent here was to be able to test the two packages. These changes result in both runtimes using `Pomelo.EntityFrameworkCore.MySql` and none using `MySql.EntityFrameworkCore`